thingsvision/core/extraction/extractors.py

```diff
@@ -394,25 +392,25 @@ def load_model_from_source(self) -> None:
         Otherwise, loads it from the cache directory.
         """
         if self.model_name in SSLExtractor.MODELS:
+
+            # unique model id name for all models
+            unique_model_filename = f'thingsvision_ssl_v0_{self.model_name}'
```

Comment:
The `unique_model_filename` no longer has an extension because it is not set in `_load_vissl_state_dict` nor in `load_state_dict_from_url`. Is this the desired behavior?

Reply:
Yes, it is better with an extension! Added it now.